code/src/components/Map.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import '@lottiefiles/lottie-player';
+
+export const Map = ({ history, gameDetails }) => {
+	const mapArray = ['0,0', '0,1', '0,2', '0,3', '1,0', '1,1', '1,2', '1,3'];
+
+	const currentLocation = gameDetails.coordinates;
+
+	const historyCoordinates = history.map((item) => item.coordinates);
+
+	return (
+		<MapContainer>
+			{mapArray.map((item, index) => (
+				<div key={index}>
+					<MapItem
+						style={
+							historyCoordinates.includes(item)
```

Comment:
Really liked the lottie-guy moving towards the correct site of the screen based on the directions. Where in the code did you define for him to move up if the direction was north, left if it was east etc?

Reply:
Right here, we made a grid of div's with flexbox mapping over each item in the mapArray, then we have the background color changing if that coordinate in the mapArray also is in the history

Reply:
Nice touch with the final room animations!